contracts/stabilizer/PegStabilityModule.sol

```diff
@@ -0,0 +1,276 @@
+pragma solidity ^0.8.4;
+
+import "./../token/Fei.sol";
+import "./../pcv/PCVDeposit.sol";
+import "./../utils/RateLimitedMinter.sol";
+import "./IPegStabilityModule.sol";
+import "./../refs/CoreRef.sol";
+import "./../refs/OracleRef.sol";
+import "../Constants.sol";
+import "@openzeppelin/contracts/security/ReentrancyGuard.sol";
+import "@openzeppelin/contracts/utils/math/SafeCast.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+abstract contract PegStabilityModule is IPegStabilityModule, CoreRef, RateLimitedMinter, OracleRef, ReentrancyGuard, PCVDeposit {
+    using Decimal for Decimal.D256;
+    using SafeCast for uint256;
+    using SafeERC20 for IERC20;
+
+    /// @notice the fee in basis points for selling asset into FEI
+    uint256 public override mintFeeBasisPoints;
+
+    /// @notice the fee in basis points for buying the asset for FEI
+    uint256 public override redeemFeeBasisPoints;
+
+    /// @notice the amount of reserves to be held for redemptions
+    uint256 public override reservesThreshold;
+
+    /// @notice the PCV deposit target
+    IPCVDeposit public override target;
+
+    /// @notice the token this PSM will exchange for FEI
+    /// This token will be set to address 0 if the bonding curve accepts eth
+    IERC20 public override token;
+
+    /// @notice FEI contract
+    IFei public FEI;
```

Comment:
This is stored in CoreRef. I know this is more gas efficient but lets not duplicate variables.